social.md

```diff
@@ -76,7 +76,7 @@ The library must implement provider callbacks for all social provider
 directories that are mapped to the specified Stormpath application.  The
 callback URLs for each provider should take the form of:
 
-> `/<stormpath.web.socialProviders.callbackRoot>/<providerId>`
+> `<stormpath.web.socialProviders.[providerId].callbackUri>/<providerId>`
```

Comment:
The dot before `.[providerId]` should be removed. And the last part (`/<providerId>`) should also be removed then :) Like this:

`<stormpath.web.socialProviders[providerId].callbackUri>`